load_data.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+###
+#
+# GTFS Manager Load Data and Rebuild OpenTripPlanner Graph object daily script
+#
+# @author LinuxWebExpert - see GitHub
+# @date 10/01/2017
+# @see https://github.com/vta/
+#
+###
+MAILTO=""
```

Comment:
:ballot_box_with_check: 
- [ ] Let's make certain `postfix` has been configured on the machine and to edit this file in production
- [ ] In staging and production, we should add a DevOps team email alias, to catch reports sent by this cronjob

See examples here - http://man7.org/linux/man-pages/man5/crontab.5.html